x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/dashboards/api/install.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IKibanaResponse, Logger, RequestHandlerContext } from '@kbn/core/server';
+import { buildRouteValidationWithZod } from '@kbn/zod-helpers';
+import { z } from '@kbn/zod';
+import { SIEM_DASHBOARD_MIGRATION_INSTALL_PATH } from '../../../../../common/siem_migrations/dashboards/constants';
+import type { SecuritySolutionPluginRouter } from '../../../../types';
+import { SiemMigrationAuditLogger } from '../../common/utils/audit';
+import { installTranslated } from './utils/installation';
+import { authz } from '../../common/utils/authz';
+import { withLicense } from '../../common/utils/with_license';
+import { withExistingDashboardMigration } from './utils/use_existing_dashboard_migration';
+
+// Define the request/response types manually since we can't regenerate them
+const InstallMigrationDashboardsRequestParams = z.object({
+  migration_id: z.string(),
+});
```

Comment:
What does that mean, can't they be imported from `x-common/siem_migrations/model/api/dashboards/dashboard_migration.gen.ts`?